cmd/main.go

```diff
@@ -230,6 +230,7 @@ func defaultNoColor() bool {
 			"GITHUB_ACTIONS",
 			"GITLAB_CI",
 			"TRAVIS",
+			"woodpecker",
```

Comment:
This will look for an environment variable `woodpecker=...`, looking at https://woodpecker-ci.org/docs/usage/environment#built-in-environment-variables I think what you want instead is to remove this line and do something like the following after the "codeship"` line below:

```go
if os.Getenv("CI_SYSTEM_NAME") == "woodpecker" {
    return false
}
```

Reply:
Thanks for the fast review. Sorry, I have not read the code carefully enough. I've changed the code to check the value of the `CI` env var now. If you prefer to use a dedicated lookup, I can also change it to:

```
if os.Getenv("CI") == "woodpecker" {
    return false
}
```